make/tools.mk

```diff
@@ -168,13 +168,24 @@ EXE_FLAGS          := $(EXE_FLAGS_EXT) $(FLAG_RELRO) -Wl,--gc-sections
 SO_FLAGS           := $(SO_FLAGS_EXT) $(FLAG_RELRO) -Wl,--gc-sections -shared $(FLAG_STDLIB) -fPIC 
 
 # Define flags for host build
-HOST_CFLAGS        := $(CFLAGS)
-HOST_CDEFS         := $(CDEFS)
-HOST_CXXFLAGS      := $(CXXFLAGS)
-HOST_CXXDEFS       := $(CXXDEFS)
-HOST_LDFLAGS       := $(LDFLAGS)
-HOST_EXE_FLAGS     := $(EXE_FLAGS)
-HOST_SO_FLAGS      := $(SO_FLAGS)
+ifeq ($(CROSS_COMPILE),1)
+  HOST_MARCH         := -march=$(shell uname -m | sed -n "s/_/-/p")
```

Comment:
I don't love this, but the other option seems to be to duplicate [this block](https://github.com/lsp-plugins/lsp-plugins/blob/master/make/system.mk#L57).

Reply:
I'm also not sure if the additional arguments (`-m32` vs `-m64` for example) are needed in some cases. Maybe it wouldn't be so bad to duplicate that block... or better yet turn it into a function if that's feasible.

Reply:
We can rewrite that block into function, I think. And reuse it. I'll try a bit later with `HOST_ARCHITECTURE` variable.